src/framework/components/collision/system.js

```diff
@@ -331,6 +333,29 @@ class CollisionMeshSystemImpl extends CollisionSystemImpl {
     // special handling
     beforeInitialize(component, data) {}
 
+    createAmmoHull(mesh, node, shape, scale) {
+        const hull = new Ammo.btConvexHullShape();
+
+        const point = new Ammo.btVector3();
+
+        const positions = [];
+        mesh.getPositions(positions);
+
+        for (let i = 0; i < positions.length; i += 3) {
+            point.setValue(positions[i] * scale.x, positions[i + 1] * scale.y, positions[i + 2] * scale.z);
+
+            // No need to calculate the aabb here. We'll do it after all points are added.
```

Comment:
Not sure this comment is valid. We don't calculate aabb inside the loop either way.

Reply:
This comment is to explain why `false` is passed as the second parameter of `addPoint`. See: https://pybullet.org/Bullet/BulletFull/btConvexHullShape_8cpp_source.html

Reply:
Ah, you are right, of course.